<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>691</height>
     </rect>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>691</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="all_tables">
       <attribute name="title">
        <string>All Tables</string>
       </attribute>
       <widget class="QTableView" name="OperationCostsTable">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>14</y>
          <width>661</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="PersonnelTable">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>220</y>
          <width>661</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="WorkAccountingTable">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>440</y>
          <width>661</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>-1</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>OperationCosts</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>205</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Personnel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>423</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>WorkAccounting</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>211</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="1">
          <widget class="QLineEdit" name="id_sklad"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Ответ</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Процедура Get price</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Номер склада:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="res_proc">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="proc">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Procedure</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Операции над таблицами</string>
        </property>
       </widget>
       <widget class="QPushButton" name="save_tables">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>103</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="refresh_tables">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>103</y>
          <width>122</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Dynamic_table">
       <attribute name="title">
        <string>Dynamic Table</string>
       </attribute>
       <widget class="QTableView" name="dynamic_table">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>901</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>440</y>
          <width>901</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="dynamic_Personnel_change">
           <property name="text">
            <string>Personnel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="dynamic_WorkAccounting_change">
           <property name="text">
            <string>WorkAccounting</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="dynamic_OperationCosts_change">
           <property name="text">
            <string>OperationCosts</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>70</width>
             <height>38</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>49</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Table:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>392</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>(none)</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="one_table">
       <attribute name="title">
        <string>VIEW</string>
       </attribute>
       <widget class="QTableView" name="view">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>901</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="task_1_sql">
       <attribute name="title">
        <string>Задача 1 SQL</string>
       </attribute>
       <widget class="QTableView" name="table_task_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>901</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>320</y>
          <width>351</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2">
          <widget class="QLineEdit" name="year">
           <property name="text">
            <string>2023</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="task_1">
           <property name="text">
            <string>Task 1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radio_task_1_SQL">
           <property name="text">
            <string>SQL</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_3</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radio_task_1_ORM">
           <property name="text">
            <string>ORM</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_3</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Год учета:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>541</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget[objectName=&quot;centralWidget&quot;]{ background-image: url(&quot;:/img/bg&quot;) no-repeat right bottom; } 
QWidget[objectName=&quot;myFrame&quot;]{ background-color: #ff0000; } 
QLineEdit{ background-color: #00ff00; } 
QWidget[enabled=&quot;false&quot;]{ background-color: #0000ff; }</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:14pt;&quot;&gt;Задача-1. Сведения об учете выработки с допускаемым браком (&amp;gt;0): номер цеха и участка; фамилия; дата; кол-во бракованных;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="task_2">
       <attribute name="title">
        <string>Задача 2</string>
       </attribute>
       <widget class="QTableView" name="window_func">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>901</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>320</y>
          <width>351</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="2">
          <widget class="QPushButton" name="task_2_buttom">
           <property name="text">
            <string>Task 2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="year_2"/>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Разряд работника:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>541</width>
          <height>111</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:12pt;&quot;&gt;Задача-2. Для каждого сотрудника вывести: ФИО, наименование профессии, разряд, количество рабочих этой профессии на этом участке, максимальный разряд рабочих этой профессии&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Задача 3</string>
       </attribute>
       <widget class="QTableView" name="table_task_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>901</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>320</y>
          <width>346</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="2">
          <widget class="QLineEdit" name="plan_qt"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="reg_qt"/>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="task_3_buttom">
           <property name="text">
            <string>Task 3</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButton_SQL">
           <property name="text">
            <string>SQL</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Количество деталей в поставке, X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Общий план, PLAN</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="radioButton_ORM">
           <property name="text">
            <string>ORM</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>541</width>
          <height>151</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:18px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Symbol';&quot;&gt;¨ &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;Задача-3. Сведения о наличии деталей на складах, такие что:&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:36px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;для этой детали &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; text-decoration: underline;&quot;&gt;имеется договорная поставка&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt; (из 09в) с планом поставки &amp;gt; $(PLAN)&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:72px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;и &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; text-decoration: underline;&quot;&gt;все поступления&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt; (из 13б) по этой договорной поставке были в количестве  &amp;gt; $(X) каждое.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="connect_page"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
